CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -6,225 +6,286 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
 and any relevant skill(s) to help you beat your opponent's guard DC.
 Melee weapons don't normally pierce armor, unless they state otherwise.
 
-- If you have Hand-to-Hand trained, +10 damage for all melee weapons and unarmed fighting.
-- Instead, if you have Hand-to-Hand Expert, +20 damage for all melee weapons and unarmed fighting. 
+If you have Hand-to-Hand trained, +10 damage for all melee weapons and 
+unarmed fighting. Instead, if you have Hand-to-Hand Expert, +20 damage for all 
+melee weapons and unarmed fighting. 
 
-- Also, If you have melee combat skill >=20 then all DEX or STR requirements for 
-  melee weapons are reduced by 1.
+Also, If you have melee combat skill >=20 then all DEX or STR requirements 
+for melee weapons are reduced by 1.
 
 (When dividing Melee Combat Skill, drop the remainder)
 
-
 ===== UNARMED FIGHTING =====
 
-----Hand-to-Hand [untrained] 	Free	----
-Damage:		    5 + STR + DEX
-Range:        	    Adjacent (1m)
+==Hand-to-Hand [untrained] Free==
+
+Damage:    5 + STR + DEX
+Range:            Adjacent (1m)
 Accuracy Modifier:  -2
 DEX Requirement:    2
 
-----Hand-to-Hand [Trained]	----
+==Hand-to-Hand [Trained]==

```

Comment:
Unambiguously could use a blank line after this subsection heading for readability.